src/App.tsx

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  message: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    message: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.key);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.key);
+  }
+
+  key = (event: KeyboardEvent) => {
+    this.setState({ message: `The last pressed key is [${event.key}]` });
```

Comment:
let's write `[space]` if the `space` is pressed
<img width="201" alt="image" src="https://user-images.githubusercontent.com/55802630/183036389-73e4fa92-898f-42a7-972f-acabeb42fd16.png">

Reply:
Fix this comment, you still have a problem with space button
![image](https://user-images.githubusercontent.com/57747481/183372360-b7f0ff37-3816-4cfc-8e00-c891c96648ed.png)